nautobot/utilities/filters.py

```diff
@@ -10,6 +10,7 @@
 from django.forms.utils import ErrorDict, ErrorList
 
 import django_filters
+from django_filters import MultipleChoiceFilter
```

Comment:
I'd prefer to skip this additional import and just reference it as `django_filters.MultipleChoiceFilter` where needed, personally.